### PR TITLE
Updated register and login page UI's

### PR DESCRIPTION
**Pull request by Sizo**
Made fields wider, replaced labels with placeholder to match android UI

### CODE REVIEW

**Comment by TheeDeafFrog** [comment]
Overall, I think it looks better, but I worry about having the description about what is in the input in the input itself: if the user enters their first name and surname, then has doubts that they entered the incorrect information in the input, then they have no way to verify that they entered in the correct place, short of refreshing the page and starting again.
There is a horizontal scroll bar showing up below it. That really needs to go.

File: Source/Web/app/Login.html [hunk 1]
```diff
@@ -28,28 +28,32 @@
...
 <!--                     Which we don't want. -->
 <!--                     <form action="" method="POST" > -->
-                        <h1>Log in to Harvest</h1>
+                        <h1><b>Log in to Harvest </b></h1>
+			<br />
                         <div class="form-group">
-                            <label for="username"><b>Email Address:</b></label>
-                            <input type="text" class="form-control" id="username" required>
+                            
+                            <input type="text" class="form-control" id="username" placeholder="username/email" required>
```
Comment (TheeDeafFrog) [inline Source/Web/app/Login.html]: Why are the placeholders here all lowercase, when some are uppercase in register page.

File: Source/Web/app/Login.html [hunk 2]
```diff
@@ -28,28 +28,32 @@
...
             <div class="row">
-                <div class="col-md-2 col-md-offset-5">
+                <div class="col-md-3  ">
                     <div class="loginBlock" align="center">                    
                     
 <!--                     We remove the form because it reloads the page -->
 <!--                     Which we don't want. -->
 <!--                     <form action="" method="POST" > -->
-                        <h1>Log in to Harvest</h1>
+                        <h1><b>Log in to Harvest </b></h1>
```
Comment (TheeDeafFrog) [inline Source/Web/app/Login.html]: Why is this bold, when it's not on the register page.

File: Source/Web/app/Register.html [hunk 3]
```diff
@@ -44,24 +45,23 @@
                         <h1>Sign up for Harvest</h1>
                         
                         <div class="form-group">
-                            <label for="name"><b>First name:</b></label>
-                            <input type="text" class="form-control" id="name" required>
+                            
+                            <input placeholder="First name:" type="text" class="form-control" id="name" required>
```
Comment (TheeDeafFrog) [inline Source/Web/app/Register.html]: The placeholder has a colon, when the rest do not.

**Comment by Letanyan** [comment]
I agree with the remark about having the description of fields inside. I've found myself forgetting what a field is because of this at times. Android is quite nice that they have it inside until you click on it then it goes outside.

**Comment by Letanyan** [comment]
Also, note we agreed that mobile designs must match the website, not the other way around.

File: Source/Web/app/Login.html (hunk 1 above)
Comment (Sizo) [reply Source/Web/app/Login.html]: Ok so based on your comments I'll revert back to labels

File: Source/Web/app/Login.html (hunk 2 above)
Comment (Sizo) [reply Source/Web/app/Login.html]: Just forgot to make the other bold. Thanks for pointing it out.

**Comment by Sizo** [comment]
Ok thanks for feedback. Ill make changes ASAP

**Pull request closed without merging**